app/actions/remote/channel.ts

```diff
@@ -1262,3 +1263,56 @@ export const handleKickFromChannel = async (serverUrl: string, channelId: string
         logDebug('cannot kick user from channel', error);
     }
 };
+
+export const getGroupMessageMembersCommonTeams = async (serverUrl: string, channelId: string) => {
```

Comment:
Since this goes to the server, let's use `fetch` instead of `get`. In general we try to have the `fetch` to go to the server and the `get` to go to the database.
```suggestion
export const fetchGroupMessageMembersCommonTeams = async (serverUrl: string, channelId: string) => {
```

Reply:
Done.